apps/web/src/components/Publication/OpenAction/UnknownModule/Tip/index.tsx

```diff
@@ -37,13 +48,27 @@ const TipOpenAction: FC<TipOpenActionProps> = ({
     return null;
   }
 
+  const hasTipped =
+    tipData?.publication?.__typename === 'Post' &&
+    Boolean(tipData.publication.operations.actedOn.length);
+
+  const countTips =
+    (tipData?.publication?.__typename === 'Post' &&
```

Comment:
Wondering why only post? why not comments, mirrors and quotes?

Reply:
In the case of a mirror we're looking at the target publication ID, so it's still be true here if the original post has tips. I don't think `Mirror` type has `operations`, actually. I forgot tipping was enabled for comments and quotes on Hey, though, so I've added those in 2a4305a